<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignIn</class>
 <widget class="QWidget" name="SignIn">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>456</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="AddrEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>356</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;HarmonyOS Sans SC&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>16</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/bin/images/btn_close_n.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="returnLogBtn">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>436</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;</string>
   </property>
   <property name="text">
    <string>返回登录</string>
   </property>
  </widget>
  <widget class="QWidget" name="loginwidget" native="true">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>26</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>116</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;HarmonyOS Sans SC&quot;;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordError">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>216</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Email">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>286</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;邮箱&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>176</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;HarmonyOS Sans SC&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Address">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>346</y>
     <width>71</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;地址&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="minisizeBtn">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>16</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/images/bin/images/btn_min_n.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="AllError">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>396</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PhoneNumEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>236</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;HarmonyOS Sans SC&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="PhoneNum">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>226</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;手机号&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Password_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>176</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;密码&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NameError">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>156</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLog">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>426</y>
     <width>21</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt ;</string>
   </property>
   <property name="text">
    <string>|</string>
   </property>
  </widget>
  <widget class="QLabel" name="LoginEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>26</y>
     <width>251</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 20pt &quot;黑体&quot;;
</string>
   </property>
   <property name="text">
    <string>系统账号注册</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="backwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>486</y>
     <width>451</width>
     <height>501</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="EmailEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>296</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;HarmonyOS Sans SC&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>436</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size: 16px;</string>
   </property>
   <property name="text">
    <string>确认注册</string>
   </property>
  </widget>
  <widget class="QLabel" name="UserName">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>HarmonyOS Sans SC</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;账号&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>NameEdit</tabstop>
  <tabstop>PasswordEdit</tabstop>
  <tabstop>PhoneNumEdit</tabstop>
  <tabstop>EmailEdit</tabstop>
  <tabstop>AddrEdit</tabstop>
  <tabstop>returnLogBtn</tabstop>
  <tabstop>createBtn</tabstop>
  <tabstop>closeBtn</tabstop>
  <tabstop>minisizeBtn</tabstop>
  <tabstop>NameError</tabstop>
  <tabstop>PasswordError</tabstop>
  <tabstop>AllError</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
